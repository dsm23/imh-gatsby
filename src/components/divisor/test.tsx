/// <reference types="jest" />

import React from "react";
import { render } from "@testing-library/react";

import Divisor from ".";

const renderable = <Divisor />;

describe(`Anchor`, () => {
  it(`it renders`, () => {
    const { baseElement } = render(renderable);

    expect(baseElement).toBeTruthy();
  });

  it(`matches snapshot`, () => {
    const { asFragment } = render(renderable);

    expect(asFragment()).toMatchSnapshot();
  });
});
