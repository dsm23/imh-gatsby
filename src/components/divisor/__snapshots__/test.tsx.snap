// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      preserveAspectRatio="xMinYMid slice"
      viewBox="0 300 1155 100"
      xmlns="http://www.w3.org/2000/svg"
    >
      <defs>
        <clippath
          id="__lottie_element_10"
        >
          <rect
            height="700"
            width="1155"
            x="0"
            y="0"
          />
        </clippath>
      </defs>
      <g
        clip-path="url(#__lottie_element_10)"
      >
        <g
          opacity="1"
          style="display: block;"
          transform="matrix(4.895999908447266,0,0,4.895999908447266,577.6270141601562,349.7330017089844)"
        >
          <g
            opacity="1"
            transform="matrix(1,0,0,1,0,0)"
          >
            <path
              d="M0 0"
              fill-opacity="0"
              stroke="rgb(181,219,255)"
              stroke-linecap="butt"
              stroke-linejoin="miter"
              stroke-miterlimit="4"
              stroke-opacity="1"
              stroke-width="0.852"
            />
          </g>
        </g>
        <g
          opacity="1"
          style="display: block;"
          transform="matrix(0.0004896000027656555,0,0,0.0004896000027656555,574.6110229492188,347.20001220703125)"
        >
          <g
            opacity="1"
            transform="matrix(1,0,0,1,0,0)"
          >
            <path
              d=" M0,-3.936500072479248 C2.1725544929504395,-3.936500072479248 3.936500072479248,-2.1725544929504395 3.936500072479248,0 C3.936500072479248,2.1725544929504395 2.1725544929504395,3.936500072479248 0,3.936500072479248 C-2.1725544929504395,3.936500072479248 -3.936500072479248,2.1725544929504395 -3.936500072479248,0 C-3.936500072479248,-2.1725544929504395 -2.1725544929504395,-3.936500072479248 0,-3.936500072479248z"
              fill="rgb(143,228,255)"
              fill-opacity="1"
            />
          </g>
        </g>
      </g>
    </svg>
  </div>
</DocumentFragment>
`;
